RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
ok maybe I'm confused and you are treating both cases (2 and PS ) modules here ?  [I'm maybe fooled by names like Phase2StripCPE which could actually be  phaseOTCPE ] ?